src/main/java/io/cryostat/net/web/http/api/v2/graph/StartRecordingOnTargetMutator.java

```diff
@@ -132,13 +133,20 @@ public HyperlinkedSerializableRecordingDescriptor getAuthenticated(
         return targetConnectionManager.executeConnectedTask(
                 cd,
                 conn -> {
-                    boolean restart = false;
+                    ReplacementPolicy replace = ReplacementPolicy.NEVER;
                     RecordingOptionsBuilder builder =
                             recordingOptionsBuilderFactory
                                     .create(conn.getService())
                                     .name((String) settings.get("name"));
-                    if (settings.containsKey("restart")) {
-                        restart = Boolean.TRUE.equals(settings.get("restart"));
+
+                    if (settings.containsKey("replace")) {
+                        String replaceValue = (String) settings.get("replace");
+                        try {
+                            replace = ReplacementPolicy.valueOf(replaceValue.toUpperCase());
+                        } catch (IllegalArgumentException e) {
+                            // Handles invalid value for "replace"
+                            replace = ReplacementPolicy.NEVER;
```

Comment:
This try-catch looks like it's doing similar or the same work as the if-else+switch in the other endpoint. Maybe the other endpoint should be written like this, too? Perhaps this logic could be extracted to a utility method, perhaps within `ReplacementPolicy`, so it can be reused?